main.cpp

```diff
@@ -31,9 +31,145 @@
 
  Wait for my code review.
  */
-
 #include <iostream>
+
+struct FloatType
+{
+    float add(float lhs, float rhs);
+    float subtract(float lhs, float rhs);
+    float multiply(float lhs, float rhs);
+    float divide(float lhs, float rhs);
+};
+
+float FloatType::add(float lhs, float rhs)
+{
+    return lhs + rhs;
+}
+
+float FloatType::subtract(float lhs, float rhs)
+{
+    return lhs - rhs;
+}
+
+float FloatType::multiply(float lhs, float rhs)
+{
+    return lhs * rhs;
+}
+
+float FloatType::divide(float lhs, float rhs)
+{
+    if (rhs == 0.0f)
+    {
+        std::cout << "Error: cannot divide by 0! Returning 0 instead\n";
+        return 0;
```

Comment:
Floating point division by 0 is legal.  

- [ ] Keep the warning, remove the `return 0;`